python/cuml/common/array.py

```diff
@@ -29,12 +31,16 @@ class Array(Buffer):
     """
     Array represents an abstracted array allocation. It can be instantiated by
     itself, creating an rmm.DeviceBuffer underneath, or can be instantiated by
-    __cuda_array_interface__ or __array_interface__ compliant, in which case
-    it'll keep a reference to that data underneath. Also can be created
+    __cuda_array_interface__ or __array_interface__ compliant arrays, in which
+    case it'll keep a reference to that data underneath. Also can be created
     from a pointer, specifying the characteristics of the array, in that case
     the owner of the data referred to by the pointer should be specified
     explicitly.
 
+    To standardize our code, please import this using:
+
+    from cuml.common.array import Array as cumlArray
```

Comment:
We should consider just renaming this Array to CumlArray if we're going to suggest importing it this way each time. Then there is no confusion and it's very transparent.

Reply:
@jakirkham @cjnolet thoughts? I'm pretty split myself on this one

Reply:
No strong thoughts.

FWIW using `Array` in cuML is analogous to using `Buffer` in cuDF.

Also most places in the Python ecosystem libraries use `ndarray`. So lower cased and prefixed. Would think risk of confusion is low (though not non-existent as John Z points out).

So it seems fine as-is, but not actually attached to that.